<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DDI::Internal::ComponentAtcEditorWidget</class>
 <widget class="QWidget" name="DDI::Internal::ComponentAtcEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="availableDrugsDb"/>
     </item>
     <item>
      <widget class="QComboBox" name="reveiwers">
       <item>
        <property name="text">
         <string>Reviewer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Eric Maeker, MD (FR)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jim Busser, MD (CA)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Graham Chui, MD (NZ)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="actionButton">
       <property name="text">
        <string>Select an action</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Load file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Search Component</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchMol"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Search ATC code</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchATC"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTableView" name="tableView">
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>253</red>
           <green>255</green>
           <blue>206</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>253</red>
           <green>255</green>
           <blue>206</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>237</red>
           <green>237</green>
           <blue>237</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="overview">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
